modules/core/package.json

```diff
@@ -72,9 +72,9 @@
     "lodash": "^4.17.14",
     "moment": "^2.20.1",
     "ripple-address-codec": "~4.1.3",
-    "ripple-binary-codec": "^0.2.4",
+    "ripple-binary-codec": "~1.1.3",
```

Comment:
Can you try `"ripple-binary-codec": "~0.2.4",` ?

Reply:
okay let me try

Reply:
Why `0.2.4`? On MS the version currently installed is 1.1.3

Reply:
@argjv Why you want to use the version 0.2.4 since its updating at the moment to the 1.1.3 wich its the stable one?